src/dune_rules/dune_file.ml

```diff
@@ -234,7 +234,25 @@ module Buildable = struct
            >>> fields
                  (field "backend"
                     (let+ libname = located Lib_name.decode
-                     and+ flags = repeat String_with_vars.decode in
+                     and+ flags =
+                       let* current_ver =
+                         Dune_lang.Syntax.get_exn Stanza.syntax
+                       in
+                       let version_check flag =
+                         let ver = (2, 8) in
+                         if current_ver < ver then
```

Comment:
Would be nice to invert this condition so that the shorter clause is at the top.

Reply:
Done.